backend/de.metas.contracts/src/main/java/de/metas/contracts/subscription/invoicecandidatehandler/FlatrateTermSubscription_Handler.java

```diff
@@ -127,65 +75,4 @@ public Quantity calculateQtyEntered(@NonNull final I_C_Invoice_Candidate icRecor
 				loadOutOfTrx(uomId, I_C_UOM.class));
```

Comment:
* u can use Quantitys.create
* or u can use UOMDAO to fetch the UOM by ID

Reply:
done.